admin/installer/test/test_cloudformation.py

```diff
@@ -0,0 +1,98 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.admin.installer.cloudformation``.
+"""
+
+from os import walk
+from subprocess import CalledProcessError
+
+from hypothesis import given
+from hypothesis.strategies import integers, one_of
+
+from twisted.python.filepath import FilePath
+
+from flocker.testtools import TestCase, run_process
+
+from .. import MIN_CLUSTER_SIZE, MAX_CLUSTER_SIZE
+
+# A Hypothesis strategy for generating supported cluster size.
+valid_cluster_size = integers(min_value=MIN_CLUSTER_SIZE,
+                              max_value=MAX_CLUSTER_SIZE)
+
+# A Hypothesis strategy for generating unsupported cluster size.
+too_small_cluster_size = integers(min_value=0,
+                                  max_value=MIN_CLUSTER_SIZE-1)
+too_big_cluster_size = integers(min_value=MAX_CLUSTER_SIZE+1,
+                                max_value=100)
```

Comment:
max_value of 100 seems rather arbitrary. I think I can see a rationale for doing
this, but I think letting hypothesis do its thing and not implying there is something
special about a size of 100 would be better.